linux_os/guide/services/ssh/ssh_server/sshd_set_max_sessions/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Set SSH MaxSessions limit'
+
+description: |-
+    The <tt>MaxSessions</tt> parameter specifies the maximum number of open sessions permitted
+    from a given connection. To set MaxSessions edit
+    <tt>/etc/ssh/sshd_config</tt> as follows: <pre>MaxSessions <b>sessions</b></pre>
```

Comment:
Use idref here: `<pre>MaxSessions <sub idref="var_sshd_max_sessions"/></pre>`

Reply:
Resolved in 0c6d42e

Reply:
This is not solved yet, change `<b>sessions</b>` to `<sub idref="var_sshd_max_sessions"/>`.

This will render the actual selected value in the profile when generating the HTML guide, and make it easy to know what the profile is setting the value to.

Reply:
Resolved in 0e95def